src/docs/sphinx/advancedExamples/validationStudies/deviatedPoroElasticWellbore/Example2.rst

```diff
@@ -0,0 +1,326 @@
+.. _AdvancedExampleDeviatedPoroElasticWellboreDrilling:
```

Comment:
This example is a fluid depletion problem with far field stresses; whereas, the DeviatedPoroElasticWellbore is a fluid injection problem without in-situ stresses, right? Is there any other difference between the two examples?

Reply:
@jhuang2601 Yes, the you are right. The first example mimic pore pressure injection problem without considering in-situ stresses and in-situ pore pressure. The second example mimic the drilling problem of a deviated wellbore in a pre-stressed and pressurized basin that leads to complex loading combinations including anisotropic compression (as considered by Detournay and Cheng 1988) and longitudinal shearing (as added by Abousleiman and Cui 1998). 

May be I will add elastic anisotropy to the second example. Analytical results exist also for the transversely isotropic case.